packages/node_modules/samples/browser-multi-party-call/test/wdio/spec/dial-and-reject.js

```diff
@@ -13,12 +12,14 @@ describe('sample-browser-multi-party-call', () => {
     const browserSpock = browser.select('browserSpock');
     const browserMccoy = browser.select('browserMccoy');
 
-    before('reload browser', () => browser.refresh());
+    before('reload browser', () => {
+      browser.select('browserSpock').reload();
```

Comment:
You have the `const`'s up above, do you need to do the select over again? Also, `reload` spawns a whole new selenium test, did you mean to do a `refresh`?

Reply:
it was mostly a matter of copying this block into each test. wasn't really paying attention to what was or was not already defined.

Yes, I'm intentionally restarting selenium. The tests were getting stuck in weird states; it about doubles the test duration, but they're way more stable.